portfolio/src/main/webapp/script.js

```diff
@@ -15,10 +15,11 @@
 /**
  * Adds a random greeting to the page.
  */
+
 function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
+    console.log("I can wiggle my ears.");
```

Comment:
I don't quite understand why this console.log statement is here.

Reply:
I had left that from the video tutorial and never got rid of it. Will take out now!